code/__DEFINES/DNA.dm

```diff
@@ -90,6 +90,8 @@
 #define NO_DNA_COPY     14
 #define DRINKSBLOOD		15
 #define NOEYES			16
+#define DYNCOLORS		17 //Use this if you want to change the race's color without the player being able to pick their own color
+#define NOHUNGER		18 //Use this if a species doesn't experience hunger in the way normal humanoids do.
```

Comment:
TRAIT_NOHUNGER already exists

Reply:
It has different behavior as far as I could tell

Reply:
It probably constantly hardsets hunger to just below well fed; you can still keep your code and replace the species trait check with a has_trait(TRAIT_NOHUNGER) check, which would work for other sources of NOHUNGER as well.

Reply:
You'll probably want to make `UNUSUALHUNGER` or something, seeing as `NOHUNGER` implies that they just bypass the system altogether.

Reply:
It does bypass the system altogether